src/lib/sftp.ts

```diff
@@ -0,0 +1,15 @@
+import { SftpClient } from "@stedi/sdk-client-sftp";
+
+let _sftpClient: SftpClient;
+
+export const sftpClient = () => {
```

Comment:
I think this whole file can be removed now, right?

Reply:
It's still used in buckets, I refactored the sftpClient out of that into it's own file

Reply:
Yeah, I'd say let's leave it in buckets and not do this refactor (it might be confusing to users -- we only really create the sftp user to get the bucket name during configuration/deployment)